src/SystemViewToolbar.tsx

```diff
@@ -0,0 +1,209 @@
+
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// React
+import React, { useState, useCallback } from "react";
+
+// Foxglove
+import Button from "@foxglove/studio-base/src/components/Button";
+import Checkbox from "@foxglove/studio-base/src/components/Checkbox";
+import ExpandingToolbar, { ToolGroup } from "@foxglove/studio-base/src/components/ExpandingToolbar";
+import FoxgloveIcon from "@foxglove/studio-base/src/components/Icon";
+import SegmentedControl, { Option } from "@foxglove/studio-base/src/components/SegmentedControl";
+import styles from "@foxglove/studio-base/src/panels/ThreeDimensionalViz/sharedStyles";
+
+// ReactFlow
+import { useStoreActions, useStoreState, useZoomPanHelper, FitViewParams } from 'react-flow-renderer';
+
+// SystemView
+import Toolbar from "./Toolbar";
+import { NodeList } from "./NodeList";
+
+// Icons
+// Used this to preview the icons: https://emotion-icons.dev/
+// import ArrowLeftRightIcon from "@mdi/svg/svg/arrow-left-right.svg";
+import {ArrowRight as ArrowLeftRightIcon} from "@emotion-icons/bootstrap/ArrowRight";
+// import ArrowUpDownIcon from "@mdi/svg/svg/arrow-up-down.svg";
+import {ArrowDown as ArrowUpDownIcon} from "@emotion-icons/bootstrap/ArrowDown";
+// import FitviewIcon from "./assets/icons/fitview.svg";
+import {PageFit as FitviewIcon} from "@emotion-icons/fluentui-system-regular/PageFit";
+// import GroupIcon from "@mdi/svg/svg/group.svg";
+import {ObjectGroup as GroupIcon} from "@emotion-icons/fa-regular/ObjectGroup";
+// import LockIcon from "@mdi/svg/svg/lock-outline.svg";
+import {Lock as LockIcon} from "@emotion-icons/bootstrap/Lock";
+// import MinusIcon from "@mdi/svg/svg/minus.svg";
+import {Minus as MinusIcon} from "@emotion-icons/boxicons-regular/Minus";
+// import PlusIcon from "@mdi/svg/svg/plus.svg";
+import {Plus as PlusIcon} from "@emotion-icons/boxicons-regular/Plus";
+// import SelectionIcon from "@mdi/svg/svg/checkbox-multiple-marked-outline.svg";
+import {SelectMultiple as SelectionIcon} from "@emotion-icons/boxicons-regular/SelectMultiple";
+// import UnlockIcon from "@mdi/svg/svg/lock-open-variant-outline.svg";
+import {Unlock as UnlockIcon} from "@emotion-icons/bootstrap/Unlock";
```

Comment:
I've replaced the icons with icons from `emotion-icons`. This seems to work with the current webpack bundler.